quintilesims/layer0
# 232 stress tests

**What does this pull request do?**
Introduces an update to the stress tests which features multi-dimensional deployment tests and  benchmarking of CLI commands under these various deployment configurations.

(This work also spawned some changes in the [tftest](https://github.com/quintilesims/tftest) package.)


**How should this be tested?**
Run `make test` in the `layer0/tests/stress/` directory (check out `layer0/tests/README.md` beforehand to make sure that you won't break things by spinning up too many resources).


**Checklist**
- ~Unit tests~
- ~Smoke tests (if applicable)~


closes  _(eventually - **TODO: that LIMITS file**)_ #232
